draft-packages/menu/KaizenDraft/Menu/Menu.tsx

```diff
@@ -45,11 +45,7 @@ export type MenuProps = GenericMenuProps & StatefulMenuProps
 type Menu = React.FunctionComponent<MenuProps>
 
 const Menu: Menu = props => {
-  const {
-    align = "left",
-    dropdownWidth = "default",
-    menuVisible = false,
-  } = props
```

Comment:
This is an unrelated change, but I removed `dropdownWidth = "default"` as it was dead code (dropdownWidth is just passed straight through to `DropdownMenu`)

Reply:
I wonder if the prop should be deprecated too? 
https://github.com/cultureamp/kaizen-design-system/pull/630/files#diff-5233a0f50609d2bf1752cd5a7f97633d1d5efe4ef48c90b946ba9b48392d64c3L22 

This is what I've been doing in components - which gives a really nice visual indicator (strikethrough) in some editors: https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/components/Button/Button.tsx#L7-L9